content/openapi/cli-plugin.md

```diff
@@ -53,6 +53,8 @@ export class CreateUserDto {
 }
 ```
 
+> info **Hint** When using `PartialType, OmitType, PickType, IntersectionType` in dtos import them from `@nestjs/swagger` instead of `@nestjs/mapped-types` for the plugin to pick up the schema
```

Comment:
```suggestion
> info **Hint** When using [mapped types utilities](https://docs.nestjs.com/openapi/mapped-types) (like `PartialType`) in DTOs import them from `@nestjs/swagger` instead of `@nestjs/mapped-types` for the plugin to pick up the schema.
```